src/components/App.tsx

```diff
@@ -38,6 +44,10 @@ const App: React.FC<AppProps> = (props) => {
     return <ErrorContainer />
   }
 
+  if (props.isMobile && props.hasStream) {
```

Comment:
We have components that are meant to be used to render components on different screen sizes in the [decentraland-ui](https://github.com/decentraland/ui/blob/master/src/components/Responsive/Responsive.tsx) repository, do any of them work for these changes?

Reply:
You seem to be checking those props to decide wether to render the `StreamContainer` or not, but the container seems to be getting those values from the state as well.